cli/cmd/check.go

```diff
@@ -50,6 +52,7 @@ func (options *checkOptions) nonConfigFlagSet() *pflag.FlagSet {
 	flags.StringVarP(&options.namespace, "namespace", "n", options.namespace, "Namespace to use for --proxy checks (default: all namespaces)")
 	flags.BoolVar(&options.preInstallOnly, "pre", options.preInstallOnly, "Only run pre-installation checks, to determine if the control plane can be installed")
 	flags.BoolVar(&options.dataPlaneOnly, "proxy", options.dataPlaneOnly, "Only run data-plane checks, to determine if the data plane is healthy")
+	flags.BoolVar(&options.multicluster, "multicluster", options.multicluster, "Run multicluster checks")
```

Comment:
just thinking out loud here (figuratively):

would it make sense to automatically enable these checks if the service mirror component has been installed?  I think it's still useful to have the command so that if the service mirror component failed to be installed for some reason, a user running `linkerd check --multicluster` would be informed of that.  But I think it might also be useful to automatically enable this flag so that users who have successfully installed the service mirror will still get informed of errors with their multicluster set up, even if they don't know (or remember) to add the `--multicluster` flag.

Reply:
Yes.. that actually makes a lot of sense. Will include that modification

Reply:
@adleong I pushed a commit that addresses that, can you take a look.